[Sprig App] Color Switcher

unknowngamer69 opened the pull request:
### Author name
Author: unknowngamer69

### About your game

**What is your game about?**
Color Switcher is a logic-based puzzle game where every move counts. You control a small player icon on a 5×5 grid made of red and blue tiles. Each time you step on a tile, it flips, and the four tiles around it also flip. Your goal is to change the entire board to a single color while using as few moves as possible.

The game features rising difficulty, smooth animations, sound effects, and a simple design that keeps each level fun and challenging.

**How do you play your game?**
Use the W, A, S, D keys to move your player around the grid. Each step flips the tile you are on and the four tiles next to it. Try to make all the tiles the same color to finish the level. Press R to restart the level anytime.

Your moves are counted, so aim to finish the puzzle with the fewest moves possible!

whatwareweb commented:
Somehow I missed that you didn't add the tags to your metadata, can you do that as shown here:
https://github.com/hackclub/sprig/blob/main/.github/PULL_REQUEST_TEMPLATE.md

unknowngamer69 commented:
hey @whatwareweb, I have fixed the errors and now i don't see any error on my editor. Also i have added tags in the Meta data.

Please review my game now

unknowngamer69 commented:
Hey @whatwareweb, its been while, when will you review this?

unknowngamer69 commented:
Hey @whatwareweb I know you are busy but will we be able to see my game getting reviewed before GTA 6? 🤧

whatwareweb commented:
OK, hello and so sorry for the delays, i was so busy :sob:

Anyway, I like the game overall and the levels are very nice and difficult as well. One thing to change is the tone that plays constantly is extremely loud, can you remove that or change it to not be so loud?

thanks!
